src/one_dragon/base/operation/application/application_group_config.py

```
@@ -38,56 +38,64 @@ def __init__(self, instance_idx: int, group_id: str):
 
     def _init_app_list(self) -> None:
         dict_list = self.get("app_list", [])
         for item in dict_list:
-            self.app_list.append(
+            self._all_apps.append(
                 ApplicationGroupConfigItem(
                     app_id=item.get("app_id", ""),
                     enabled=item.get("enabled", False),
                 )
             )
+        # 初始化时 app_list 包含全部，update_full_app_list 调用后才过滤
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)